source/fab/tools/tool.py

```diff
@@ -63,7 +63,8 @@ def check_available(self) -> bool:
         :returns: whether the tool is working (True) or not.
         '''
         try:
-            self.run(self._availability_option)
+            op = self._availability_option
+            self.run(op)
```

Comment:
Why has an intermediate variable been added?

Reply:
Absolutely no idea ;) Maybe I had some debug print in between? I've reverted that.